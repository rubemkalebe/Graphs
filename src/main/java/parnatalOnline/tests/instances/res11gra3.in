11 3
25 24 14 16 17 13 1 15 12 16
13 15 8 9 4 7 4 25 19
15 7 10 11 18 6 13 24
1 20 4 9 5 23 13
14 4 2 3 5 21
6 29 8 27 16
8 27 10 1
6 9 30
24 12
9
