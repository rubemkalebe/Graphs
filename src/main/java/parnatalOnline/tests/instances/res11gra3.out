35.000000
0 7, 1 6, 1 8, 2 8, 3 4, 3 8, 4 7, 4 9, 5 6, 6 10, 
1359666000
2141.369602;0.044013;